engine/src/stack3.cpp

```diff
@@ -136,6 +136,12 @@ IO_stat MCStack::extendedload(MCObjectInputStream& p_stream, uint32_t p_version,
 	return t_stat;
 }
 
+IO_stat MCStack::load(IO_handle stream, uint32_t version)
+{
+	MCLog("%s should never be called!", __PRETTY_FUNCTION__);
+	MCUnreachable();
+}
+
 IO_stat MCStack::load(IO_handle stream, uint32_t version, uint1 type)
```

Comment:
I thought we had discussed renaming this method as it is not the same semantics as load(). (Refrains from ranting about why function name overloading is actually a bad thing for code readability and comprehension ;))